monkestation/code/modules/security/code/weapons/lawbringer.dm

```diff
@@ -367,7 +399,10 @@
 		var/atom/throw_target = get_edge_target_turf(target, angle2dir(Angle))
 		target.throw_at(throw_target, 4, 1)
 
-
+/**
+ * lawbringer tideshot mode:
+ * EXTREMELLY fucks up assistants. stuns them, applies negative mood to them and more
```

Comment:
Better description would be “Taser that only effects assistants”

Reply:
fair enough, changed